lms/static/sass/partials/base/_variables.scss

```diff
@@ -372,7 +372,7 @@ $body-line-height: golden-ratio(.875em, 1) !default;
 $base-font-color: rgb(60,60,60) !default;
 $baseFontColor: $base-font-color;
 
-$lighter-base-font-color: rgb(100,100,100) !default;
+$lighter-base-font-color: rgb(100,100,100) $base-font-color;
```

Comment:
I did not hand-edit these reverts. I simply did the git command to revert a commit, so the new code is what was there before (for better or worse!).